// stringcase/shim/node/index.js
/**
 * Convert string cases between camel case, pascal case, snake case etc...
 * @module stringcase
 */
'use strict';

var camelcase = require('./camelcase');

var capitalcase = require('./capitalcase');

var constcase = require('./constcase');

var cramcase = require('./cramcase');

var decapitalcase = require('./decapitalcase');

var dotcase = require('./dotcase');

var enumcase = require('./enumcase');

var lowercase = require('./lowercase');

var pascalcase = require('./pascalcase');

var pathcase = require('./pathcase');

var sentencecase = require('./sentencecase');

var snakecase = require('./snakecase');

var spacecase = require('./spacecase');

var spinalcase = require('./spinalcase');

var titlecase = require('./titlecase');

var trimcase = require('./trimcase');

var uppercase = require('./uppercase');

exports.camelcase = camelcase;
exports.capitalcase = capitalcase;
exports.constcase = constcase;
exports.cramcase = cramcase;
exports.decapitalcase = decapitalcase;
exports.dotcase = dotcase;
exports.enumcase = enumcase;
exports.lowercase = lowercase;
exports.pascalcase = pascalcase;
exports.pathcase = pathcase;
exports.sentencecase = sentencecase;
exports.snakecase = snakecase;
exports.spacecase = spacecase;
exports.spinalcase = spinalcase;
exports.titlecase = titlecase;
exports.trimcase = trimcase;
exports.uppercase = uppercase;
module.exports = {
  camelcase: camelcase,
  capitalcase: capitalcase,
  constcase: constcase,
  cramcase: cramcase,
  decapitalcase: decapitalcase,
  dotcase: dotcase,
  enumcase: enumcase,
  lowercase: lowercase,
  pascalcase: pascalcase,
  pathcase: pathcase,
  sentencecase: sentencecase,
  snakecase: snakecase,
  spacecase: spacecase,
  spinalcase: spinalcase,
  titlecase: titlecase,
  trimcase: trimcase,
  uppercase: uppercase
};
